gwctl/cmd/describe.go

```diff
@@ -96,6 +96,23 @@ func runDescribe(cmd *cobra.Command, args []string, params *utils.CmdParams) {
 			}
 		}
 		policiesPrinter.PrintDescribeView(policyList)
+	
+	case "policycrd", "policycrds":
+		var policyCrdList []policymanager.PolicyCRD
+		if len(args) == 1 {
+			policyCrdList = params.PolicyManager.GetCRDs()
+		} else {
+			var found bool
+			policyCrd, found := params.PolicyManager.GetCRD(args[1])
```

Comment:
For users, args[1] would usually mean the actual "name" of the CRD. It's worth noting here that the "name" of the CRD is not the `policyCrdID` which is being used as the "key" for the policyCRDs map within `manager.go`.

This means that if someone does `gwctl describe policycrds mycrd.foo.bar`, it might not work.

I think we need to modify the `GetCRD` function to accept a `crdName` and then manually search for the CRD in the map and return the right one.

Reply:
Yes I agree with this. I will make the changes